src/error.rs

```diff
@@ -2,6 +2,7 @@ use std::ffi::c_int;
 use std::ffi::c_uint;
 use strum::FromRepr;
 
+#[allow(clippy::upper_case_acronyms)]
```

Comment:
Can you make this an `expect` with a `reason` (the reason being these are the same names as the C errors)?